modules/dists/BlockDist.chpl

```diff
@@ -1641,90 +1494,76 @@ proc BlockArr.doiUseBulkTransferStride(B) {
   // If multi-ddata is possible then we can only do strided bulk
   // transfer when all the blocks have but a single ddata chunk.
   //
+  if this.rank != B.rank then return false;
   return defRectSimpleDData
-         || (oneDData && B._value.oneDData);
+         || (oneDData && chpl__getActualArray(B).oneDData);
 }
 
-proc BlockArr.doiBulkTransfer(B) {
+proc BlockArr.doiBulkTransfer(B, viewDom) {
   if debugBlockDistBulkTransfer then
     writeln("In BlockArr.doiBulkTransfer");
+  const actual = chpl__getActualArray(B);
+  // TODO: how to avoid a deep copy of the domain here? If it's distributed,
+  // it can cause a lot of comms...
+  pragma "no copy" const actDom = chpl__getViewDom(B);
```

Comment:
I find this pragma "no copy" suspect and the comment next to it too.
The reason is that I wouldn't expect
  const actdom = chpl__getViewDom(B)
to create any copy at all in the calling code. If there is any copy made, it would be within chpl__getViewDom, in which case the pragma "no copy" would have no effect.

Does it work to simply remove this pragma?